ImputationPipeline/ScoringPart.wdl

```diff
@@ -304,47 +305,124 @@ task AdjustScores {
 
 			population_data = merge(population_pcs, population_scores, by.x="IID", by.y="X.IID")
 
-			# generate the linear model from the population data using the first 5 PCs
-			population_model = glm(SCORE1_SUM ~ PC1 + PC2 + PC3 + PC4 + PC5, data = population_data, family = "gaussian")
+			# generate the linear model from the population data using the first 4 PCs
+			population_model = glm(SCORE1_SUM ~ PC1 + PC2 + PC3 + PC4, data = population_data, family = "gaussian")
 
 			population_data$residual_score = resid(population_model)
+			population_data$residual_score2 = resid(population_model)^2
 			population_resid_mean = mean(population_data$residual_score)
```

Comment:
population_resid_mean and population_resid_sd are not used anywhere?

Reply:
good catch, no they are revenants from the old code.  will remove.

Reply:
same with population_data$residual_score (unsquared)

Reply:
same with population_data$residual_score